components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/fragments/policy-add/public/js/index.js

```diff
@@ -49,6 +55,7 @@ var addPolicy = function () {
         "id": 0,
         "enabled": false,
         "description": "",
+        "roleList" : "Internal/everyone",
```

Comment:
Is there a reason to define this in a js file? Can't we get this form back end and use it here?